backend/danswer/configs/app_configs.py

```diff
@@ -135,7 +135,7 @@
     os.environ.get("POSTGRES_PASSWORD") or "password"
 )
 POSTGRES_HOST = os.environ.get("POSTGRES_HOST") or "localhost"
-POSTGRES_PORT = os.environ.get("POSTGRES_PORT") or "5432"
+POSTGRES_PORT = os.environ.get("POSTGRES_PORT") or "5433"
```

Comment:
looks like local dev changes got into the PR across a few files

Reply:
Yep! Meant to keep this as a draft for now - still in progress